daemon/graphdriver/aufs/aufs.go

```diff
@@ -264,6 +265,40 @@ func (a *Driver) createDirsFor(id string) error {
 	return nil
 }
 
+// Helper function to debug EBUSY errors on remove.
+func debugEBusy(mountPath string) (string, error) {
+	// lsof is not part of GNU coreutils. This is a best effort
+	// attempt to detect offending processes.
+	c1 := exec.Command("lsof")
+	c2 := exec.Command("grep", mountPath)
+	var err error
+
+	c2.Stdin, err = c1.StdoutPipe()
+	if err != nil {
+		return "", fmt.Errorf("Assigning pipes failed with %v", err)
+	}
+
+	if err := c1.Start(); err != nil {
+		return "", fmt.Errorf("Starting %s failed with %v", c1.Path, err)
+	}
+
+	var pipeErr error
+	defer func() {
+		pipeErr = c1.Wait()
```

Comment:
Nothing seems to be using this error and the `if` in line290 can never be non-nil.